.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,7 +8,19 @@ For new features contributions please make sure you have completed the following
 essential items:
 
 * [ ] `CHANGELOG.md` updated to document new feature additions
-* [ ] Appropriate documentation updates in the [docs][]
+* [ ] Appropriate documentation updates in the [docs repo]()
```

Comment:
This change will break the link.

Reply:
At the bottom of the file we have the reference for the link:

`[docs]: https://opentelemetry.io/docs/demo/`